components/Navigator/components/Content/index.jsx

```diff
@@ -1,48 +1,41 @@
-import React, { Fragment } from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { INDEX_PATH } from '@shopgate/pwa-common/constants/RoutePaths';
-import connect from './connector';
 import Logo from './components/Logo';
 import Title from './components/Title';
 import Search from './components/Search';
 
 /**
- * The navigator content component.
- * @param {Object} props The component props.
- * @returns {JSX}
+ * The NavigatorContent component.
  */
-const Content = (props) => {
-  let currentTitle = null;
+class NavigatorContent extends Component {
```

Comment:
Make it at least a `PureComponent`. You only have primitive props.

Reply:
Done